x-pack/plugins/infra/public/components/empty_states/no_remote_cluster.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useLinkProps } from '@kbn/observability-plugin/public';
+import { NoIndices } from './no_indices';
+
+export const NoRemoteCluster = () => {
+  const settingLinkProps = useLinkProps({ app: 'metrics', pathname: '/settings' });
+
+  const goToSettings = (
+    <EuiButton data-test-subj="infraHostsPageGoToSettings" color="danger" {...settingLinkProps}>
+      {i18n.translate('xpack.infra.hostsPage.goToMetricsSettings', {
+        defaultMessage: 'Go to Metrics Settings',
+      })}
+    </EuiButton>
+  );
+
+  return (
+    <NoIndices
+      color="danger"
+      iconType="error"
+      titleSize="m"
+      title={i18n.translate('xpack.infra.sourceConfiguration.noRemoteClusterTitle', {
+        defaultMessage: 'Remote Cluster Unavailable',
```

Comment:
> Suggesting this wording because the remote cluster might be available, if the issue is from the config, we just couldn’t connect to it.

Makes total sense, thanks for checking!